trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.length > 10 ? @nameable.correct_name[0...10] : @nameable.correct_name
```

Comment:
**OPTIONAL** - According to the project requirements, this method should trim the name if the length of the name is more than `10`. I suggest you fix the upper boundary to `9` since ruby is zero-indexed. Please fix this 😄